package.json

```diff
@@ -24,4 +24,4 @@
     "typescript": "~5.0.4"
   },
   "private": true
-}
+}
```

Comment:
Revert changes to this file

Reply:
@schaudhari6254888, please revert changes here